src/irr/asset/CGraphicsPipelineLoaderMTL.cpp

```diff
@@ -90,6 +90,18 @@ SAssetBundle CGraphicsPipelineLoaderMTL::loadAsset(io::IReadFile* _file, const I
     {
         auto layout = makePipelineLayoutFromMtl(materials[i]);
         pipelines[i] = core::make_smart_refctd_ptr<ICPURenderpassIndependentPipeline>(nullptr, std::move(layout), nullptr, nullptr, vtxParams, blendParams, primParams, rasterParams);
+        if (materials[i].maps[CMTLPipelineMetadata::SMtl::EMP_OPACITY].size())
+        {
+            for (uint32_t i = 0u; i < SBlendParams::MAX_COLOR_ATTACHMENT_COUNT; ++i)
+            {
+                blendParams.blendParams[i].blendEnable = true;
+                blendParams.blendParams[i].srcColorFactor = EBF_SRC_ALPHA;
+                blendParams.blendParams[i].srcAlphaFactor = EBF_SRC_ALPHA;
+                blendParams.blendParams[i].dstColorFactor = EBF_ONE_MINUS_SRC_ALPHA;
+                blendParams.blendParams[i].dstAlphaFactor = EBF_ONE_MINUS_SRC_ALPHA;
+            }
```

Comment:
wait what!? why all attachments? you're only gonna write to the first one with this pipeline!

Reply:
https://github.com/buildaworldnet/IrrlichtBAW/pull/410#discussion_r368315402